core/shell.py

```diff
@@ -548,8 +548,28 @@ def Main(lang, arg_r, environ, login_shell, loader, readline):
   assert home_dir is not None
   rc_path = flag.rcfile
   # mycpp: rewrite of or
+
+  rc_paths = []
+  # TODO: Decide if this should be first or last rc_path (as in: should it overwrite modular includes or not)
   if rc_path is None:
-    rc_path = os_path.join(home_dir, '.config/oil/%src' % lang)
+    rc_paths.append(os_path.join(home_dir, '.config/oil/%src' % lang))
+  else:
+    rc_paths.append(rc_path)
+
+  # Experimentally try to load all files in ~/.config/oil/oshrc.d | ~/.config/oil/oilrc.d
+  # TODO: Decide on proper directory, `oil/oshrcd.d` seems ugly
+  # TODO: Add a flag/parameter to manually pass/disable reading the rc_dir
```

Comment:
Yes, see `frontend/flag_def.py` 

Please add `--rcdir`, which is like `--rcfile`

This will help us test it!